docs/swagger.yaml

```diff
@@ -2557,7 +2557,7 @@ paths:
             * pacing: Course pacing. Possible values: instructor, self
             * tabs: Course tabs
             * user_timezone: User's chosen timezone setting (or null for browser default)
-            * can_load_course: Whether the user can view the course (AccessResponse object)
+            * can_load_courseware: Whether the user can view the course (AccessResponse object)
```

Comment:
Does this still need to be in the swagger docs here? It looks like this isn't returned at all anymore.

Reply:
No, it does not